crates/sovereign-sdk/rollup-interface/src/state_machine/zk/mod.rs

```diff
@@ -80,6 +76,9 @@ pub trait Zkvm: Send + Sync {
         serialized_proof: &[u8],
         code_commitment: &Self::CodeCommitment,
     ) -> Result<StateTransition<Da, Root>, Self::Error>;
+
+    /// Return the code commitment for the zkVM program
+    fn get_code_commitment(&self) -> Self::CodeCommitment;
```

Comment:
This one is not implemented for bonsai, so what's the point?

Reply:
it is 

```rust
    fn get_code_commitment(&self) -> Self::CodeCommitment {
         self.image_id
     }
```